src/e2e-test/resources/pluginParameters.properties

```diff
@@ -18,7 +18,7 @@ invalid.admin.consumer.secret=lmnop891011
 #SOQL Query
 simple.query=SELECT Id, Name, Phone FROM Account
 test.query=SELECT Id,Name,Col_Timestamp__c,Col_Date__c,Col_Currency__c,Col_Email__c,Col_Number__c,\
-  Col_GeoLocation__Latitude__s,Col_GeoLocation__Longitude__s,Col__c,Col_Url__c,Col_Time__c,Col_Text__c FROM Automation_custom__c
+ Col__c,Col_Url__c,Col_Time__c,Col_Text__c FROM Automation_custom__c
```

Comment:
Is this change done to fix the broken environment dependent tests?

Reply:
Yes, we have done this change to fix the broken environment dependent tests.